src/lib/arrays.js

```diff
@@ -0,0 +1,119 @@
+const getNthElement = (index, array) => {
+  return array[index % array.length];
+};
+
+const arrayToCSVString = array => {
+  return array.join(',');
+};
+
+const csvStringToArray = string => {
+  return string.split(',');
+};
+
+const addToArray = (element, array) => {
+  return array.push(element);
+};
+
+const addToArray2 = (element, array) => {
+  return array.concat(element);
+};
+
+const removeNthElement = (index, array) => {
+  if (!index) {
+    return array.shift();
+  }
+  return array.splice(index, index - 1);
+};
+
+const numbersToStrings = numbers => {
+  return numbers.map(String);
+};
+
+const uppercaseWordsInArray = strings => {
+  return strings.map(words => words.toUpperCase());
+};
+
+const reverseWordsInArray = strings => {
+  strings.map((element, index) => {
+    element = element.split('');
+    element = element.reverse();
+    element = element.join('');
+    strings[index] = element;
+  });
+  return strings;
+  /*
+  const newArray = []
+  strings.forEach(element => {
+    element = element.split('');
+    element = element.reverse();
+    element = element.join('');
+    newArray.push(element);
+  });
+  return newArray;
+  */
+};
+
+function onlyEven(numbers) {
+  const evenNumbers = numbers.filter(element => {
+    if (element % 2 === 0) {
+      return element;
+    }
+  });
+  console.log(evenNumbers);
+  return evenNumbers;
+}
+
+const removeNthElement2 = (index, array) => {
+  const items = array.filter(element => {
+    if (array[index] !== element) {
+      return element;
+    }
+  });
+  return items;
```

Comment:
Good to leave a clear line above the final return statement for multiline functions